WindowsServerDocs/remote/remote-desktop-services/rds-vdi-recommendations-2004.md

```diff
@@ -0,0 +1,794 @@
+
+- Microsoft: [Removing Windows 10 in-box apps during a task sequence](https://blogs.technet.microsoft.com/mniehaus/2015/11/11/removing-windows-10-in-box-apps-during-a-task-sequence/)
+- Microsoft: [Removing Built-in apps from Windows 10 WIM-File with PowerShell - Version 1.3](https://gallery.technet.microsoft.com/Removing-Built-in-apps-65dc387b)
+- Microsoft: [Windows 10 1607: Keeping apps from coming back when deploying the feature update](https://blogs.technet.microsoft.com/mniehaus/2016/08/23/windows-10-1607-keeping-apps-from-coming-back-when-deploying-the-feature-update/)
+- Microsoft: [Removing Windows 10 in-box apps during a task sequence](https://blogs.technet.microsoft.com/mniehaus/2015/11/11/removing-windows-10-in-box-apps-during-a-task-sequence/)
+
+Then run the Remove-AppxProvisionedPackage PowerShell command to remove UWP app payloads is as follows:
+
+```powershell
+Remove-AppxProvisionedPackage -Online - PackageName MyAppxPackage 
+```
+
+As a final note on this topic, each UWP app should be evaluated for applicability in each unique environment. You will want to install a default installation of Windows 10 2004, then note which apps are running and consuming memory. For example, you may want to consider removing apps that start automatically, or apps that automatically display information on the Start Menu, such as Weather and News, and that may not be of use in your environment.
+
+> [!NOTE] 
+> If utilizing the scripts from GitHub, you can easily control which apps are removed before running the script. After downloading the script files, locate the file ‘AppxPackage.json’, edit that file, and remove entries for apps that you want to keep, such as Calculator, Sticky Notes, etc.
+
+
+## Windows Optional Features cleanup
+
+### Managing Optional Features with PowerShell
+
+Microsoft: [https://social.technet.microsoft.com/wiki/contents/articles/39386.windows-10-managing-optional-features-with-powershell.aspx](https://social.technet.microsoft.com/wiki/contents/articles/39386.windows-10-managing-optional-features-with-powershell.aspx)
+
+You can manage Windows Optional Features using PowerShell. To enumerate currently installed Windows Features, run the following PowerShell command:
+
+```powershell
+Get-WindowsOptionalFeature -Online
+```
+
+Using PowerShell, an enumerated Windows ZOptional Feature can be configured as enabled or disabled, as in the following example:
+
+```powershell
+Enabled-WindowsOptionalFeature -Online -FeatureName "DirectPlay" -All
+```
+
+Examples of disabling features in the virtual desktop image:
+
+```powershell
+Disable-WindowsOptionalFeature -Online -FeatureName "WindowsMediaPlayer"
+```
+
+Next, you may want to remove teh WIndows Media Player package.
+
+```powershell
+
+PS C:\> Get-WindowsPackage -Online -PackageName *media*
+
+PS C:\Windows\system32> Get-WindowsPackage -Online -PackageName *media*       
```

Comment:
Is this duplication of the command necessary, or can we do with just the "Run as Administrator" version (2nd line)?

Reply:
Removed the duplicated line